<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0">
  <channel>
    <title>OpenAI 开发者论坛 - 最新帖子</title>
    <link>https://community.openai.com</link>
    <description>最新帖子</description>
    <lastBuildDate>Fri, 10 May 2024 12:33:41 GMT</lastBuildDate>
    <item>
      <title>制作风格化 T 恤图像设计提示</title>
      <link>https://community.openai.com/t/crafting-stylized-t-shirt-image-design-prompts/662503#post_7</link>
      <description><![CDATA[我只是尝试重新创建没有 T 恤的图像
提示：
“创建一个图形设计，其中一个人从后面，穿着连帽衫，站在一条小巷里，周围是高楼大厦和明亮的霓虹灯标志。场景应该大气，通过细致的艺术作品和光线的运用来捕捉城市生活的本质。将设计框在粗边框内。从设计中排除任何文本或品牌元素，使其适合按需打印。”
]]></description>
      <guid>https://community.openai.com/t/crafting-stylized-t-shirt-image-design-prompts/662503#post_7</guid>
      <pubDate>Fri, 10 May 2024 12:32:48 GMT</pubDate>
    </item>
    <item>
      <title>目前人工智能如何集成到健身追踪设备和应用程序中？</title>
      <link>https://community.openai.com/t/how-is-artificial-intelligence-currently-being-integrated-into-fitness-tracking-devices-and-apps/741786#post_1</link>
      <description><![CDATA[人工智能通过提供个性化见解、预测分析和实时反馈，正在彻底改变健身跟踪设备和应用程序。这些设备利用人工智能算法来分析来自加速度计和心率监视器等传感器的数据，以准确跟踪活动。此外，人工智能还有助于解释用户行为模式、建议量身定制的锻炼计划、预测受伤风险并优化表现。总体而言，人工智能集成增强了用户体验，激发了更健康的习惯，并促进了更有效的健身制度。]]></description>
      <guid>https://community.openai.com/t/how-is-artificial-intelligence-currently-being-integrated-into-fitness-tracking-devices-and-apps/741786#post_1</guid>
      <pubDate>Fri, 10 May 2024 12:31:25 GMT</pubDate>
    </item>
    <item>
      <title>即使创建新帐户，API 也会给出配额错误，并且在检查使用情况时，所有 5 美元都完好无损</title>
      <link>https://community.openai.com/t/api-giving-quota-error-even-when-new-account-is-created-and-on-checking-the-usage-got-all-5-intact-there/741692#post_3</link>
      <description><![CDATA[这意味着 openai 不再提供免费的 5 美元，因为我已经使用它很长一段时间了，并且之前它可以工作？]]></description>
      <guid>https://community.openai.com/t/api-giving-quota-error-even-when-new-account-is-created-and-on-checking-the-usage-got-all-5-intact-there/741692#post_3</guid>
      <pubDate>Fri, 10 May 2024 12:31:06 GMT</pubDate>
    </item>
    <item>
      <title>ChatGPT 生产力影响研究</title>
      <link>https://community.openai.com/t/research-on-productivity-impact-of-chatgpt/741778#post_1</link>
      <description><![CDATA[您好，我正在尝试查找有关 ChatGPT 对人们生产力影响的研究，有人知道我在哪里可以找到这样的研究吗？]]></description>
      <guid>https://community.openai.com/t/research-on-productivity-impact-of-chatgpt/741778#post_1</guid>
      <pubDate>Fri, 10 May 2024 12:22:45 GMT</pubDate>
    </item>
    <item>
      <title>用于防止即时注入和越狱的 API</title>
      <link>https://community.openai.com/t/api-to-prevent-prompt-injection-jailbreaks/203514?page=2#post_29</link>
      <description><![CDATA[耶！我们终于得到了。这是缺失的部分。
def fetch_run_output(thread_id, run_id):
    提取值=无
    而真实：
        run_status = client.beta.threads.runs.retrieve(thread_id=thread_id, run_id=run_id)
        if run_status.status in [&#39;已完成&#39;, &#39;失败&#39;, &#39;已取消&#39;]:
            print(&quot;运行状态：&quot;, run_status.status)
            休息
        print(&quot;运行仍在处理...&quot;)
        时间.睡眠(2)

    如果 run_status.status == &#39;已完成&#39;:
        消息 = client.beta.threads.messages.list(thread_id=thread_id)
        对于 messages.data 中的消息：
            print(&quot;消息来自:&quot;, message.role)
            if message.role == &#39;assistant&#39; and message.content: # 假设系统消息包含数据
                对于 message.content 中的 content_block：
                    如果 content_block.type == &#39;文本&#39;:
                        # 正确访问嵌套结构中的“值”
                        提取值=内容块.文本.值
                        print(&quot;提取的内容值：&quot;, extract_value)
                        休息
    别的：
        print(&quot;运行未成功完成：&quot;, run_status.last_error)
        返回无

    返回提取值

控制台输出：
[TextContentBlock(text=Text(annotations=, value=&#39;法式吐司&#39;), type=&#39;text&#39;) ]
仍在运行处理...
运行状态：已完成
留言来自：助理
提取的内容值：{“卡路里”：“126-154”}
消息来自：用户
助手已删除：AssistantDeleted(id=‘asst_’,deleted=True,object=‘assistant.deleted’)
基本上，步骤如下

创建助理。从此处捕获 Assistant_id。
创建一个线程。从这里捕获 thread_id。
现在为 thread_id 创建一条消息。这是用户输入的地方。对我们来说这是一个菜名。例如。法式吐司。
现在，为 thread_id 和 Assistant_id 创建一个 Run。捕获run_id。
现在您必须根据该 run_id 检查 run_status，并且只有当 run_status 完成时，任务才完成且输出可行。参见上面的方法。
工作完成后删除助手。这是可选的，但对我们有用。
]]></description>
      <guid>https://community.openai.com/t/api-to-prevent-prompt-injection-jailbreaks/203514?page=2#post_29</guid>
      <pubDate>Fri, 10 May 2024 12:17:24 GMT</pubDate>
    </item>
    <item>
      <title>错误 异常活动 GPT4 WEB</title>
      <link>https://community.openai.com/t/error-unusual-activity-gpt4-web/736901#post_9</link>
      <description><![CDATA[此处相同，在自定义 GPT 中
]]></description>
      <guid>https://community.openai.com/t/error-unusual-activity-gpt4-web/736901#post_9</guid>
      <pubDate>Fri, 10 May 2024 12:08:30 GMT</pubDate>
    </item>
    <item>
      <title>ChatGPT 第 1 层限制，错误 429</title>
      <link>https://community.openai.com/t/chatgpt-limit-of-tier-1-with-error-429/741763#post_1</link>
      <description><![CDATA[你好，
我目前正在使用 GPT-4 Vision Preview Tier 1。据我了解，一旦每日请求 (RPD) 达到 10,000，我预计会遇到 429 错误，表明已超出限制。我的疑问是，在这种情况下如何无缝过渡到第 2 层？具体来说，如果我在达到 10,000 个请求后遇到此错误，我应该采取什么步骤自动切换到第 2 层？或者如何撤销以不更改 TIER？]]></description>
      <guid>https://community.openai.com/t/chatgpt-limit-of-tier-1-with-error-429/741763#post_1</guid>
      <pubDate>Fri, 10 May 2024 12:08:26 GMT</pubDate>
    </item>
    <item>
      <title>新闻文章：中国因“政治宣传”而封锁 ChatGPT</title>
      <link>https://community.openai.com/t/news-article-china-blocks-chatgpt-due-to-political-propganda/72941#post_9</link>
      <description><![CDATA[现在 OpenAI 已经屏蔽了像我这样使用 VPN 连接的用户。如果没有VPN，中国人甚至看不到OpenAI的页面或谷歌在中国的东西。现在，我们使用ChatGPT的最后一种方式已经消失了。
]]></description>
      <guid>https://community.openai.com/t/news-article-china-blocks-chatgpt-due-to-political-propganda/72941#post_9</guid>
      <pubDate>Fri, 10 May 2024 12:06:48 GMT</pubDate>
    </item>
    <item>
      <title>GPT4 “嗯……好像出了什么问题。”</title>
      <link>https://community.openai.com/t/gpt4-hmm-something-seems-to-have-gone-wrong/669247?page=3#post_48</link>
      <description><![CDATA[今天也一样，联系了支持团队并收到了一条 chatgpt 类型的消息。我已完成指示的所有操作（清除缓存、清除历史记录、禁用扩展），但没有任何更改。]]></description>
      <guid>https://community.openai.com/t/gpt4-hmm-something-seems-to-have-gone-wrong/669247?page=3#post_48</guid>
      <pubDate>Fri, 10 May 2024 12:02:42 GMT</pubDate>
    </item>
    </channel>
</rss>